CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # changelog
 
+## 6.4.3 - 15 Feb 2017
+
+* Create a wrapper for `listBuildTargets` in `Client.js`.
+* Marked `compileCode`, `signalDevice`, `listDevices` and `listBuildTargets` as deprecated. Those methods will be removed in 6.5
```

Comment:
if we can keep these methods, but re-implement them in terms of the new model then we don't have to deprecate them. e.g.  `compileCode`, `signalDevice`, `listDevices` etc all work well as is.